domains/eren.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "erenizm",
+  "repo": "https://github.com/erenndev",
+  "owner": {
+    "username": "Eren",
+    "email": "erenizm1903@gmail.com"
+  },
+  "record": {
+    "URL": "https://erenizm.github.io"
```

Comment:
For you to use a github.io subdomain, it has to be your username, so in this case it should be erenndev.github.io, also I think you might want to use CNAME considering you might want people to visit eren.is-a.dev instead of erenndev.github.io

Reply:
OK, I now edit file